### PR TITLE
improve: ils & retire: Add Top10 Mappings, ils add example alert

### PR DESCRIPTION
**Pull request by kingthorin**
- Add Top 10 mappings.
- Update build files with commonlib dependency.
- Update unit tests.
- Update changelogs.
- ILS add example alert.

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: addOns/imagelocationscanner/src/main/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRule.java [hunk 1]
```diff
@@ -171,6 +163,30 @@ public String getAuthor() {
...
+                .setEvidence(getAlertDetailPrefix()  + "\n" + gpsDetails)
+                .setCweId(200) // CWE-200: Information Exposure
+                .setWascId(13); // WASC-13: Information Leakage
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        List<Alert> alerts = new ArrayList<>();
+        String gpsDetails = "\n  Location:: \n"
+                + "    Exif_GPS: 40° 50' 19\", -74° 12' 33\"";
```
Comment (thc202) [inline addOns/imagelocationscanner/src/main/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRule.java:183]: Could be a single line (same comment for the following statement).

File: addOns/imagelocationscanner/src/test/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRuleUnitTest.java [hunk 2]
```diff
@@ -64,6 +64,7 @@ void passesIfExifLocationDetected() throws HttpMalformedHeaderException, IOExcep
         // Then
         assertEquals(alertsRaised.size(), 1);
         validateAlert(alertsRaised.get(0));
+        System.out.println(alertsRaised.get(0).getEvidence());
```
Comment (thc202) [inline addOns/imagelocationscanner/src/test/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRuleUnitTest.java:67]: Remove.

File: addOns/imagelocationscanner/src/test/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRuleUnitTest.java [hunk 3]
```diff
@@ -109,6 +110,7 @@ void passesIfPrivacyExposureDetected() throws HttpMalformedHeaderException, IOEx
...
```
Comment (thc202) [inline addOns/imagelocationscanner/src/test/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRuleUnitTest.java:113]: Remove.

File: addOns/imagelocationscanner/imagelocationscanner.gradle.kts [hunk 4]
```diff
@@ -10,13 +10,24 @@ zapAddOn {
     manifest {
         author.set("Jay Ball (veggiespam) and the ZAP Dev Team")
         url.set("https://www.zaproxy.org/docs/desktop/addons/image-location-and-privacy-scanner/")
+
+    dependencies {
```
Comment (thc202) [inline addOns/imagelocationscanner/imagelocationscanner.gradle.kts:14]: Should be moved to the right.

File: addOns/imagelocationscanner/src/test/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRuleUnitTest.java [hunk 5]
```diff
@@ -61,6 +64,7 @@ void passesIfExifLocationDetected() throws HttpMalformedHeaderException, IOExcep
...
```
Comment (psiinon) [inline addOns/imagelocationscanner/src/test/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRuleUnitTest.java:67]: This really needed?

File: addOns/imagelocationscanner/src/test/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRuleUnitTest.java [hunk 6]
```diff
@@ -106,6 +110,7 @@ void passesIfPrivacyExposureDetected() throws HttpMalformedHeaderException, IOEx
...
```
Comment (psiinon) [inline addOns/imagelocationscanner/src/test/java/org/zaproxy/zap/extension/imagelocationscanner/ImageLocationScanRuleUnitTest.java:113]: Ditto

**Review by psiinon — COMMENTED** [review]
All good apart from the System.out.printlns :)

**Comment by kingthorin** [comment]
Thank you both, will tweak that shortly.

**Comment by kingthorin** [comment]
Should I flatten or leave as two commits?

**Comment by thc202** [comment]
I'm fine with two commits.

**Comment by kingthorin** [comment]
Done

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**